packages/rspack/src/lib/cache/MemoryCachePlugin.ts

```diff
@@ -0,0 +1,55 @@
+import type { Compiler } from "../../Compiler";
+import { Cache } from "../Cache";
+
+export default class MemoryCachePlugin {
+	static PLUGIN_NAME = "MemoryCachePlugin";
+	apply(compiler: Compiler) {
+		const cache: Map<
+			string,
+			{
+				etag: string | null;
+				data: any;
```

Comment:
can we use `unknown` here? it is better than `any`

Reply:
fixed, copied from webpack type doc